src/avatar.jsx

```diff
@@ -156,7 +154,7 @@ const Avatar = React.createClass({
           className={className}
         >
           {icon && React.cloneElement(icon, {
-            color: color,
+            color: color || avatar.color,
             style: Object.assign(styles.icon, icon.props.style),
```

Comment:
I guess it'd be too much to ask users to apply custom style with fill for `SvgIcons` and color for `FontIcons`. :laughing: